test/integration/targets/git/tasks/checkout-new-tag.yml

```diff
@@ -48,6 +48,9 @@
     that:
       - not update_new_tag|changed
       - "'newtag' in listoftags.stdout_lines"
+  # This test fails on osx.
```

Comment:
@robinro Could you take a look at this test to see why it's not passing on OS X? If you need access to a system for testing, let me know.